src/app/pages/cars/cars/cars-list-table-data-source.ts

```diff
@@ -64,7 +65,15 @@ export class CarsListTableDataSource extends TableDataSource<Car> {
     return new Observable((observer) => {
       // Get cars
       this.centralServerService.getCars(this.buildFilterValues(), this.getPaging(), this.getSorting()).subscribe((cars) => {
-        this.createAction.visible = cars.canCreate;
+        // Initialize authorization actions
+        this.authorizationActions = {
+          canCreate: Utils.convertToBoolean(cars.canCreate),
+          canListUsers: Utils.convertToBoolean(cars.canListUsers),
+          canListCarCatalog: Utils.convertToBoolean(cars.canListCarCatalog),
+          canCreatePoolCar: Utils.convertToBoolean(cars.canCreatePoolCar)
+        };
+        // Update filters visibility
+        this.updateFilterVisibilityWithAthorizationActions();
```

Comment:
No need a method here I think or add the same kind of method in all the table data source :).